integr8ly/installation
# removed wt2 route creation

## Additional Information
https://issues.jboss.org/browse/INTLY-3063
## Verification Steps
Jenkins pipeline with this feature branch was executed here: https://integreatly-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/pds-install/1350/console and it was confirmed that no wt2 routes are present in 3scale namespace. Walkthrough 2 was also completed successfully with an evals user on this cluster: https://master.ppaszki-2f75.open.redhat.com

If you however want to verify the issue yourself repeat the steps, e.g.:

- install integreatly with this branch
- confirm that there are no wt2 routes in 3scale namespace
- complete wt2 successfully

### Installation Verification
- Ensure the author of the PR has attached a log of the installation run from his branch to the jira or pr and check that it exited as expected.
- Verify the fresh installation is correct on cluster provided by PR author 
### Upgrade Verification
Does not affect upgrade

## Is an upgrade task required and are there additional steps needed to test this?



installation log: https://gist.github.com/pawelpaszki/0181189392e1f5108b45220868c205ba


3scale routes: 

<img width="1581" alt="Screenshot 2019-10-04 at 11 35 49" src="https://user-images.githubusercontent.com/11087605/66201431-43017880-e69b-11e9-9303-dbb187da719c.png">
